src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js

```diff
@@ -90,11 +90,17 @@ class StatusForm extends React.Component { // eslint-disable-line react/prefer-s
 
   handleSubmit = e => {
     e.preventDefault();
-    this.props.onPatchIncident({
-      id: this.props.incident.id,
-      type: 'status',
-      patch: { status: { state: this.form.value.status, text: this.form.value.text } },
-    });
+    const { value } = this.form;
+
+    if (value.text.match(/{{[^}]+}}/gi) !== null) {
```

Comment:
I would suggest a RegEx like this: `/(?:{{|}})/gi`

And replace the `.match()` for `.test()`

```js
if (value.text.test(/(?:{{|}})/gi)
```

Reply:
Indeed, much simpler.